utils/common/src/main/kotlin/Extensions.kt

```diff
@@ -161,12 +160,12 @@ fun File.safeDeleteRecursively(force: Boolean = false, baseDirectory: File? = nu
  *
  * @throws IOException if any missing directory could not be created.
  */
-fun File.safeMkdirs() {
+fun File.safeMkdirs(): File {
```

Comment:
Maybe add it to the documentation that the created `File` is returned.

Reply:
Just a hint: it may be that multiple files (dirs) get created.